tests/unit_tests/test_controller_utils.py

```diff
@@ -86,6 +86,7 @@ def _check_controller_resources(
         infra = config.pop('infra')
         assert infra in expected_infra_list
         expected_infra_list.remove(infra)
+        config.pop('_no_missing_accel_warnings')
```

Comment:
This seems wrong - we shouldn't be getting the `_no_missing_accel_warnings` in the YAML because it should be none. If the test is failing I think that's a bug.